pastis/plotting.py

```diff
@@ -994,3 +995,130 @@ def natural_keys(text):
     https://stackoverflow.com/questions/5967500/how-to-correctly-sort-a-string-with-a-number-inside)
     """
     return [atoi(c) for c in re.split(r'(\d+)', text)]
+
+
+def plot_thermal_mus(mus, nmodes, nsegments, c0, out_dir, save=False):
+    """
+    Generates modal constraints plot for individual segment.
+
+    Parameters
+    ----------
+    mus : ndarray
+        list of standard deviations for each segment
+    nmodes : int
+        number of thermal modes
+    nsegments :  int
+        number of segments
+    c0 : scalar
+        target contrast
+    out_dir : str
+        path to save the plot
+    save : bool
+        whether to save the plot
+    """
+    harris_coeffs_table = np.zeros([nmodes, nsegments])
+    for qq in range(nmodes):
+        for kk in range(nsegments):
+            harris_coeffs_table[qq, kk] = mus[qq + (kk) * nmodes]
```

Comment:
Remove redundant prentheses.

Is this double-loop just rearanging the loaded data? If so, please make a little function out of it that we can reuse, since this is likely something we will be needing all over the place.